### ISSUE TITLE
EfficientNet: sync batch norm during pretrain?

### ISSUE THREAD

**Issue by eric-haibin-lin** [tensorflow/tpu#806]
EfficientNet-L2 is pretty large - how is the batch normalization layer handled during training? How many images are used for doing batch norm? It seems that 1 image per core is used - does that mean some kind of synchronized batch normalization is used on TPU? Thanks in advance! @mingxingtan

**Comment by mingxingtan** [comment]
Hi @eric-haibin-lin It is using sync batch-norm, which aggregates the batch mean/variance of all TPU cores.

**Comment by eric-haibin-lin** [comment]
thanks for the quick reply!
